src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Quotas & Events'
+sidebar_order: 0
+---
+
+Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events to be tracked. This number of events is your quota. When an event is accepted by Sentry, it counts toward your quota.
+
+Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.
+
+## Key Terms
+
+Let’s clarify a few terms to start:
+-   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a unique set of characteristics, called its fingerprint.
+-   Issue - an issue is a grouping of similar events, who all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
+-   Quota - your quota is the monthly number of events you pay Sentry to track.
+
+## What Counts Toward My Quota, an Overview
+
+Sentry completes a thorough evaluation of each event to determine if it counts toward your quota, as outlined in this overview. Detailed documentation for each evaluation is linked throughout. Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed and contains valid fingerprint information; if any of these items are missing or incorrect, the event is rejected.
+
+1. **SDK configuration**
+
+      The SDK configuration either allows the event or filters the event out. For more information, see [Outbound Filters in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#outbound-filters) or [Filtering Events]({%- link _documentation/error-reporting/configuration/index.md -%}).
```

Comment:
Did you mean to link to `/error-reporting/configuration/filtering.md`?

Reply:
Yes. Changed and tested the link. Thank you.